frontend/src/welcome-panel/WelcomePanel.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import './WelcomePanel.less';
+
+interface WelcomePanelProps {
+  onHide(e: React.MouseEvent): void;
+}
+
+const WelcomePanel: React.StatelessComponent<WelcomePanelProps> = ({ onHide }) => (
+  <div className='WelcomePanel welcome-panel'>
+    <a
+      className='welcome-panel-close'
+      href='#'
+      onClick={onHide}
+    >
```

Comment:
@Ishirak It's not the same because that other example could not reasonably fit in a single line while this one could. But again, it's a small thing, either style is OK and will not make a huge difference to readability.